__tests__/app.test.js

```diff
@@ -162,7 +162,43 @@ describe('/api/articles/:article_id', () => {
 });
 
 describe('Post article comment', () => {
-    test('should post the article', () => {
-        
+    test('should post the article and be successful', () => {
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send( {
+            body: "Wordsssssssssssss",
+            username: "butter_bridge",
+        })
+        .expect(200)
+        .then(({body: { comment }}) => {
+            expect(comment).toEqual(expect.objectContaining({
+                comment_id: 19,
+                body: 'Wordsssssssssssss',
+                article_id: 1,
+                author: 'butter_bridge',
+                votes: 0,
+                created_at: expect.any(String)
+            }))
+        });
+    });
+
+    test('should fail with 400 when given an empty object ', () => {
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send({})
+        .expect(400)
+        .then((responce) => {
+            expect(responce.res.statusMessage).toEqual('Bad Request');
+        });
+    });
+
+    test('should fail with 400 when given the wrong items in the object ', () => {
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send({ rating_out_of_five: 6})
+        .expect(400)
+        .then((responce) => {
+            expect(responce.res.statusMessage).toEqual('Bad Request');
```

Comment:
We are missing a few tests:
- 200 ignores unnecessary properties
- 400 article_id is not a number
- 404 article_id does not exist
- 404 username is not found